src/pages/ResourceBatchUpdatePage.tsx

```diff
@@ -84,19 +84,25 @@ const ResourceBatchUpdatePage = ({
 
   // page constants
   const pageSize = 10;
-  const resourceCount = targetResourceData?.targetResources?.length || 0;
+  const resourceCount = targetResourceData?.targetResources?.length ?? 0;
   const pageCount = Math.ceil(resourceCount / pageSize);
   const mainResourceName = targetResourceData?.mainResourceName;
-  const language = contextLanguage || Language.FI;
+  const language = contextLanguage ?? Language.FI;
   const targetResourcesStorageKey = 'targetResources';
+  // forms a nice string from feedback emails env variable
+  const feedbackEmailsString = window?.ENV?.FEEDBACK_EMAILS?.replace(/\s/g, '')
+    .replace(/,(?=[^,]*$)/, ' ja ')
+    .replace(/,/g, ', ');
```

Comment:
Would it be easier to split the variable `window?.ENV?.FEEDBACK_EMAILS.split(',')` and use them `Lähetä virheilmoituksesta ruutukaappaus sähköpostitse ${feedbackEmailsString[0]} ja ${feedbackEmailsString[1]}` 🤔

Reply:
Well... We don't maybe actually know how many emails there are or even just one (or 2, three...)?